app/assets/stylesheets/typography.scss

```diff
@@ -3,7 +3,8 @@
 
 // Alternate base fonts
 body.sans-serif-article-body {
-  .body {
+  .body,
```

Comment:
These changes seem related to a post and not the editor. If so this could be another PR.

Reply:
This is tightly related to this PR since it's for post preview in the editor.